plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -153,10 +153,19 @@ export function handleConfigQuery(
     Array.isArray(queryConfig.values)
   ) {
     try {
-      span.setAttribute(
-        AttributeNames.PG_VALUES,
-        JSON.stringify(queryConfig.values)
-      );
+      const convertedValues = queryConfig.values.map(value => {
+        if (!value) {
+          return 'null';
+        } else if (value instanceof Buffer) {
```

Comment:
```suggestion
        } else if (Buffer.isBuffer(value)) {
```

Reply:
Why prefer `isBuffer()` over `instanceof`? it seems in the code that it's the same: 
https://github.com/nodejs/node/blob/main/lib/buffer.js#L556

Reply:
ah ok. Feel free to keep it as is. I was assuming that the existence of `Buffer.isBuffer()` indicates that there is some more magic included there.